.github/workflows/checks.yaml

```diff
@@ -41,7 +41,7 @@ jobs:
       - name: Set up JDK
         uses: actions/setup-java@5896cecc08fd8a1fbdfaf517e29b571164b031f7
         with:
-          java-version: "11"
+          java-version: "17"
```

Comment:
Gateway is currently at `11` so I think we need to live with the terrible strings for now.

Reply:
rolled back to java 11